robottelo/decorators/func_locker.py

```diff
@@ -41,16 +41,14 @@ def test_that_conflict_with_test_to_lock(self)
 """
 import functools
 import inspect
-import logging
 import os
 import tempfile
 from contextlib import contextmanager
 
 from pytest_services.locks import file_lock
 
 from robottelo.config import settings
-
-logger = logging.getLogger('robottelo')
+from robottelo.logging import logger
```

Comment:
should this be a logzero import?

Reply:
I'd prefer for modules within the framework to import the logger directly from `robottelo.logging` - that way if any further changes are made to the logger, or it gets replaced, the framework imports don't change.

Reply:
good call